### PR TITLE
fix: update naming

### PR DESCRIPTION
**Pull request by wyuinche**
### Pull-request Type (fix, feat, bug, doc, chore, test, etc...) 

- fix

### Current Behavior (Link to an open issue)

- old operation naming style

### New Behavior (if this is a feature change)

- update names of operations

### CODE REVIEW

File: cmds/design.go [hunk 1]
```diff
@@ -84,12 +85,12 @@ func (akd *AccountKeysDesign) IsValid([]byte) error {
 	return nil
 }
 
-type GenesisCurrenciesDesign struct {
+type RegisterGenesisCurrencyDesign struct {
```
Comment (soonkuk) [inline cmds/design.go]: @wyuinche The type "RegisterGenesisCurrencyDesign" actually refers to the CurrencyDesign for the genesis operation. Thus, GenesisCurrencyDesign would be a more appropriate name.

**Review by soonkuk — CHANGES_REQUESTED** [review]

File: cmds/design.go (hunk 1 above)
Comment (wyuinche) [reply cmds/design.go]: @soonkuk I couldn't find what this struct is used for. What name should it be?

Comment (soonkuk) [reply cmds/design.go]: @wyuinche I suggests "GenesisCurrencyDesign".

Comment (wyuinche) [reply cmds/design.go]: @soonkuk It has been modified.

**Review by soonkuk — APPROVED** [review]

**Pull request merged**